crates/router/src/services/oidc_provider.rs

```diff
@@ -53,3 +64,250 @@ pub async fn get_jwks(state: SessionState) -> RouterResponse<&'static JwksRespon
         })
         .map(ApplicationResponse::Json)
 }
+
+pub fn validate_authorize_params(
```

Comment:
Nit: Can we update the visibility of functions to keep them private by default, public (or `pub(crate)`) only if necessary?